examples/substrate-to-evm-fungible-transfer/package.json

```diff
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "@buildwithsygma/sygma-sdk-core": "2.2.0",
-    "@polkadot/api": "10.7.3",
+    "@polkadot/api": "10.7.2",
```

Comment:
Why didn't we downgrade this?

Reply:
this was downgraded to match the version that was used in the rest of the SDK.

I tried updating to a newer version than 10.7.2 but there were some type incompatibilities that prevented the project from building.